Makefile

```diff
@@ -91,6 +91,11 @@ ENABLE_NBGL_QRCODE = 1
 #ENABLE_NBGL_KEYBOARD = 1
 #ENABLE_NBGL_KEYPAD = 1
 
+########################################
+#       SWAP FEATURE      			   #
+########################################
+ENABLE_SWAP = 1
```

Comment:
I suggest we don't give device lib rights to the application
We can finish and use this PR to do so 
https://github.com/LedgerHQ/ledger-secure-sdk/pull/872

Reply:
Enforced by the new ENABLE_TESTING_SWAP flag